File: bronze/5597/c++/main.cpp
#include <iostream>

int main()
{
    int array[30], num, resert[2], count = 0;
    for (int i = 0; i < 30; i++) {
        array[i] = i +1;
    }
    for (int i = 0; i < 28; i++) {
        std::cin >> num;
        array[num -1] = 0;
    }
    
    for (int data : array) {
        if (data != 0) {
            resert[count++] = data;
        }
    }
    if (resert[0] > resert[1]) {
        std::cout << resert[1] << '\n' << resert[0];
    } else {
        std::cout << resert[0] << '\n' << resert[1];
    }
    return 0;
}

File: bronze/5597/python/main.py

if __name__ == "__main__":
    resert = list()
    for i in range(30):
        resert.append(i +1)
    for _ in range(28):
        num = int(input())
        resert[num -1] = 0
    a, b, c = set(resert)
    if (b > c):
        print(c)
        print(b)
    else:
        print(b)
        print(c)